lib/cc/yaml/nodes/glob.rb

```diff
@@ -2,8 +2,11 @@ module CC
   module Yaml
     module Nodes
       class Glob < Scalar
-        def match?(path)
-          File.fnmatch(to_s, path)
+        DOUBLE_STAR_END = Regexp.new('\/\*\*$').freeze
+        DOUBLE_STAR_BEGIN = Regexp.new('\*\*\.').freeze
+
+        def value
+          @value.sub(DOUBLE_STAR_END, "/**/*").sub(DOUBLE_STAR_BEGIN, "**/*.")
```

Comment:
Did you spend and time trying to do this with a single `sub`? Now that we have all those (great) test cases, I might much around with it a little bit :)